### ISSUE TITLE
[Bug] FT.AGGREGATE's SORTBY narg is always off by 1

### ISSUE THREAD

**Issue by allada** [aembke/fred.rs#281]
When `FT.AGGREGATE` command is being dispatched, `SORTBY` has the syntax:
`SORTBY {nargs} {property} {ASC|DESC} [MAX {num}]`

The documentation explicitly calls out:
```
nargs is the number of sorting parameters, including ASC and DESC, for example, SORTBY 4 @foo ASC @bar DESC.
```
https://redis.io/docs/latest/commands/ft.aggregate/

If you run the following snippet, it'll generate the following:
```
 Array([String("FT.AGGREGATE"), String("test"), String("@test:{ x\\:* }"), String("LOAD"), String("1"), String("data"), String("SORTBY"), String("1"), String("@test"), String("ASC"), String("WITHCURSOR")])
 ```
 
 the command:
`String("SORTBY"), String("1"), String("@test"), String("ASC")`
should be:
`String("SORTBY"), String("2"), String("@test"), String("ASC")`

snippet:
```rs
use fred::prelude::*;

#[tokio::main]
async fn main() -> Result<(), RedisError> {
use tracing_subscriber::{EnvFilter, fmt, prelude::*};

tracing_subscriber::registry()
    .with(fmt::layer())
    .with(EnvFilter::from_default_env())
    .init();

  let client = RedisClient::default();
  client.init().await?;

  let foo: u64 = client.ft_aggregate(
    "test",
    "@test:{ x\\:* }",
    fred::types::FtAggregateOptions {
        load: Some(fred::types::Load::Some(vec![fred::types::SearchField {
            identifier: "data".into(),
            property: None,
        }])),
        cursor: Some(fred::types::WithCursor {
            count: None,
            max_idle: None,
        }),
        pipeline: vec![
            fred::types::AggregateOperation::SortBy  {
                properties: vec![(
                    "@test".into(),
                    fred::types::SortOrder::Asc
                )],
                max: None,
            }
        ],
        ..Default::default()
    })
    .await.unwrap();

  Ok(())
}

```

**Comment by allada** [comment]
Solution is simple:
https://github.com/aembke/fred.rs/blob/7ccef4c5032c1806d93666beeddd872ea0e91a68/src/commands/impls/redisearch.rs#L114

```diff
diff --git a/src/commands/impls/redisearch.rs b/src/commands/impls/redisearch.rs
index 176f7d7..641e7fe 100644
--- a/src/commands/impls/redisearch.rs
+++ b/src/commands/impls/redisearch.rs
@@ -111,7 +111,7 @@ fn gen_aggregate_op(args: &mut Vec<RedisValue>, operation: AggregateOperation) -
       args.extend([static_val!(APPLY), expression.into(), static_val!(AS), name.into()]);
     },
     AggregateOperation::SortBy { properties, max } => {
-      args.extend([static_val!(SORTBY), properties.len().try_into()?]);
+      args.extend([static_val!(SORTBY), properties.len().try_into()? * 2]);
       for (property, order) in properties.into_iter() {
         args.extend([property.into(), order.to_str().into()]);
       }

```

**Comment by aembke** [comment]
Thanks @allada, this will be fixed in 9.1.2 - https://github.com/aembke/fred.rs/pull/282

**Comment by aembke** [comment]
Should be fixed in [9.1.2](https://crates.io/crates/fred/9.1.2)
